spec/unit/persistence/active_record_persistence_spec.rb

```diff
@@ -399,6 +399,39 @@
     expect(persistor).not_to be_sleeping
   end
 
+  describe 'pessimistic locking' do
+    let(:worker) { Worker.create!(:name => 'worker', :status => 'sleeping') }
+
+    subject { transactor.run! }
+
+    context 'no lock' do
+      let(:transactor) { NoLockTransactor.create!(:name => 'no_lock_transactor', :worker => worker) }
+
+      it 'should invoke pessimsitic lock with true' do
```

Comment:
@TheDudeWithTheThing and @alto, I've corrected the spec descriptions in a  follow up commit. Thanks for the feedback.